Exercise_1.java

```diff
@@ -1,8 +1,6 @@
 //Time complexity- 0(n)

```

Comment:
Think again? Please read more on runtime complexity for every operation in Stack.